k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/ImapResponseParser.java

```diff
@@ -412,6 +411,22 @@ private String parseQuoted() throws IOException {
         throw new IOException("parseQuoted(): end of stream reached");
     }
 
+    private String parseQuotedOrNil() throws IOException {
```

Comment:
any reason why this one didn't get a `Nullable` annotation but the other method did?